### PR TITLE
Pair-programming

### PR DESCRIPTION
**Pull request by jlberbesi**
For this project I have:

- Implemented only a basic UI with plain HTML.
- Created a collection that keeps a list of books.
- Created a function to add a new book to the collection, with title and author.
- Created a function to remove a book from the collection (hint: you can use the [array filter()]

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+function displayBooks() {
+  // Clear the current book list
+  bookList.innerHTML = '';
+
+  // Add each book to the list
+  books.forEach((book, index) => {
+    const li = document.createElement('li');
+    li.innerText = `${book.title} by ${book.author}`;
+    bookList.appendChild(li);
```
Comment (m05e5) [inline script.js:15]: - [ ] Please make sure to have your app running properly. Books are not being added due to an error with the bookList
image

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far. !
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve
- [ ] Please make sure your readme file is professional and follows microverse Readme template
- [ ] Please make sure to have a professional PR description that includes the changes you applied
- [ ] Please check the comments below 

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (jlberbesi) [reply script.js:15]: @m05e5 Thanks. I highly appreciate  your feedback provided at this point. I'll work on those suggestions then.

**Review by jasonfelice — APPROVED** [review]
Hi @jlberbesi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<p align="center"></p>

## Highlights
No Linter errors✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**